### DISCUSSION TITLE
Regen énergie

### DISCUSSION THREAD

**Discussion by Voisin1234** [Crownicles/Crownicles#2389] [Nouvelles idées]
Bonsoir. Actuellement, lorsque nous montons de niveau, nos statistiques s'améliorent, et notre vie est restaurée au maximum. Je suggère simplement d'ajouter que les points de PV (énergie) soient également remis à 100 %.

**Comment by BastLast** [comment]
C'est le cas tous les 10 levels

**Reply by Voisin1234** [reply]
Comme pour 10 20 etc ?

**Reply by Voisin1234** [reply]
Je suis passé 70, et ça ne l’a pas fait.
![IMG_7372](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/108148891/468c719c-d67f-423e-bde4-9f17ce61b7ab)
![IMG_7374](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/108148891/76fc3025-0246-423a-bd2e-1dbbc600d86f)

**Reply by BastLast** [reply]
Bah si ?

**Reply by BastLast** [reply]
Ah mais tu parle de l'énergie ?

**Reply by Voisin1234** [reply]
Oui Désolé, si je me suis mal fait comprendre

**Reply by BastLast** [reply]
> Oui Désolé, si je me suis mal fait comprendre

Si tu peix edit le titre pr que ca soit plus clair ca serait top

**Reply by Voisin1234** [reply]
C'est fais désolé pour la confusion.
